assets/grafana/celeborn-dashboard.json

```diff
@@ -410,17 +396,17 @@
                   }
                 ]
               },
-              "unit": "none"
+              "unit": "bytes"
             },
             "overrides": []
           },
           "gridPos": {
             "h": 8,
             "w": 12,
-            "x": 0,
+            "x": 12,
```

Comment:
The newly added panels are arranged left-aligned and can be adjusted to two in a row.
The demo I saw here using the new template is like below:
![image](https://github.com/apache/incubator-celeborn/assets/19429353/43f6d76e-3520-48a7-a17c-534ce7249d4e)